tests/x11regressions/evolution/evolution_meeting_pop.pm

```diff
@@ -0,0 +1,276 @@
+        send_key "alt-n";
+    };
+    sleep 1;
+    type_string "$mail_user";
+    # add self-signed CA with internal account
+    if ($i =~ m/internal/) {
+        assert_and_click "evolution_wizard-sending-checkauthtype";
+        assert_screen "evolution_mail_meeting_trust_ca";
+        send_key "alt-a";
+        wait_screen_change {
+            send_key "$next";
+            send_key "ret";
+        }
+    }
+    else {
+        send_key "$next";
+    }
+    if (sle_version_at_least('12-SP2')) {
+        send_key "ret";    #only in sp2, send ret to next page
+    }
+    assert_screen "evolution_wizard-account-summary";
+    send_key "$next";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "alt-n";    #only in sp2
+        send_key "ret";
+    }
+    assert_screen "evolution_wizard-done";
+    send_key "alt-a";
+
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-a";    #disable keyring option, only in SP2
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-init-window") {
+        send_key "super-up";
+    }
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    assert_screen "evolution_mail-max-window";
+}
+
+sub send_meeting_requst {
+
+    my ($self, $sender, $receiver, $mail_subject) = @_;
+    my $config      = $self->getconfig_emailaccount;
+    my $mail_box    = $config->{$receiver}->{mailbox};
+    my $mail_passwd = $config->{$sender}->{passwd};
+
+    #create new meeting
+    send_key "shift-ctrl-e";
+    assert_screen "evolution_mail-compse_meeting", 30;
+    send_key "alt-a";
+    sleep 2;
+    type_string "$mail_box";
+    send_key "alt-s";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "alt-s";    #only need in sp2
+    }
+    type_string "$mail_subject this is a pop test meeting";
+    send_key "alt-l";
+    type_string "the location of this meetinng is conference room";
+    assert_screen "evolution_mail-compse_meeting", 60;
+    send_key "ctrl-s";
+    assert_screen "evolution_mail-sendinvite_meeting", 60;
+    send_key "ret";
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-a";    #disable keyring option, only need in SP2 or later
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    assert_screen "evolution_mail-compse_meeting", 60;
+    send_key "ctrl-w";
+    if (check_screen "evolution_mail-save_meeting_dialog") {
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-send_meeting_dialog") {
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-meeting_error_handle") {
+        send_key "alt-t";
+    }
```

Comment:
Instead of wasting time with all these singular `check_screen` better do an `assert_screen` with multiple tags and use `match_has_tag`.